src/components/invite-dialog/index.tsx

```diff
@@ -45,7 +45,7 @@ export class InviteDialog extends React.Component<Properties, State> {
   }
 
   get inviteText() {
-    return this.props.inviteCode;
+    return `Use this code to join me on ZERO Messenger: ${this.props.inviteCode} https://zos.zer0.io/get-access`;
```

Comment:
We definitely shouldn't hard code the url. Do we not have env vars for the root url?

Reply:
We'll have to add a new one for the root url

Reply:
Seems like we have REACT_APP_MESSENGER_INVITE_PATH

Reply:
Sure, I'll update.

Reply:
Updated to use REACT_APP_MESSENGER_INVITE_PATH & removed the env var I added to github variables. Changes merged :)